### ISSUE TITLE
depandancy on lodash package

### ISSUE THREAD

**Issue by mshanak** [rosmo/sails-arangodb#12]
hello 
you should mention in read-me to install "lodash" package using this command
`npm install lodash`

if not installed the following error will appear 

```
/home/mahdi/apps/app1/node_modules/sails-arangodb/lib/processor.js:8
  this.schema = _.cloneDeep(schema);
                  ^
TypeError: _.cloneDeep is not a function
```

**Comment by gabriel-letarte** [comment]
Looking into it
Is this happening on `master` branch or `convert-to-4.0.1` branch?

Thanks

**Comment by gabriel-letarte** [comment]
I just merged `convert-to-4.0.1` to `master` and deployed to NPM under version `0.2.1`.
Could you test again?

Thanks

**Comment by gabriel-letarte** [comment]
I closed this PR since I had no feedback.
Let me know if the problem persist but I am pretty sure it has been fixed by `0.2.1+`

**Comment by ziaongit** [comment]
I had the same issue
